.devcontainer/devcontainer.json

```diff
@@ -3,16 +3,16 @@
     "build": {
         "dockerfile": "Dockerfile",
         "args": {
-            "SWIFT_VERSION" : "5.9",
+            "SWIFT_VERSION" : "6.1",
             "HYLO_LLVM_BUILD_TYPE": "${localEnv:HYLO_LLVM_BUILD_TYPE:MinSizeRel}"
-	}
+        }
     },
     "features": {
         "ghcr.io/devcontainers/features/common-utils:2": {
             "installZsh": "false",
             "username": "vscode",
-            "userUid": "1000",
-            "userGid": "1000",
+            "userUid": "1001",
+            "userGid": "1001",
```

Comment:
Good idea.  I know why you did this; it's usually prudent to add a comment for subtle things like this. `// userid 1001 becayse https://something…` could be sufficient.